### PR TITLE
Remove `github.com/pkg/errors` since it's not maintained anymore.

### PR DESCRIPTION
**Pull request by MustafaSaber**
Updates https://github.com/sarslanhan/cronmask/issues/2

### CODE REVIEW

File: cron.go [hunk 1]
```diff
@@ -110,28 +111,28 @@ func parseCronField(fieldIdx int, fieldStr string) (field, error) {
...
-			return nil, errors.Errorf("could not parse the cron field: %s. invalid list item: %s", fieldStr, p)
+			return nil, fmt.Errorf("could not parse the cron field: %s. invalid list item: %s", fieldStr, p)
 		}
 		possibleRangeFields := strings.Split(p, "-")
 
 		if len(possibleRangeFields) == 1 {
 			parsed, err := parseValue(fieldIdx, possibleRangeFields[0])
 			if err != nil {
-				return nil, errors.Wrapf(err, "could not parse cron field: %s. invalid list item: %s", fieldStr, p)
+				return nil, fmt.Errorf("%w could not parse cron field: %s. invalid list item: %s", err, fieldStr, p)
```
Comment (AlexanderYastrebov) [inline cron.go:121]: This should be
```
fmt.Errorf("could not parse cron field: %s. invalid list item %q: %w", fieldStr, p, err)
```
Original lib puts err to the end of the message, see https://go.dev/play/p/xr6ev6Q1Hyt

File: go.mod [hunk 2]
```diff
@@ -1,5 +1,3 @@
 module github.com/sarslanhan/cronmask
 
 go 1.12
```
Comment (AlexanderYastrebov) [inline go.mod:3]: Lets also bump this to 1.19

**Comment by AlexanderYastrebov** [comment]
:+1:

**Review by sarslanhan — APPROVED** [review]
LGTM too. Thanks folks

**Pull request merged**